admin/routes/restart.js

```diff
@@ -5,10 +5,12 @@ const router = express.Router()
 
 const {
   getRestartOverview,
-  getSelectRestartList
+  getSelectRestartList,
+  postSubmitRestartList
 } = require('../controllers/restart')
 
 router.get('/overview', isAuthenticated(config.ROLE_TEACHER), (req, res) => getRestartOverview(req, res))
 router.get('/select-restart-list', isAuthenticated(config.ROLE_TEACHER), (req, res, next) => getSelectRestartList(req, res, next))
+router.post('/submit-restart-list', isAuthenticated(config.ROLE_TEACHER), (req, res, next) => postSubmitRestartList(req, res, next))
```

Comment:
would it be better convention to prefix with the feature, so it would be /restart-submit-list etc? 🤔

Reply:
@GuyHarwood as pointed on the previous comment it's going to be:
/restart/submit-restart-list